qemu/tests/cfg/qemu_guest_agent.cfg

```diff
@@ -8,6 +8,7 @@
     gagent_status_cmd = "service qemu-guest-agent status"
     gagent_pkg_check_cmd = "rpm -q qemu-guest-agent"
     Windows:
+        username = "Administrator"
```

Comment:
This conflicts with another param, so please rename it.

Reply:
update, replace username with ga_username